### DISCUSSION TITLE
Modifying parameters of C-Rate by changing electrode size. OKane / NMC811

### DISCUSSION THREAD

**Discussion by KarlMur** [pybamm-team/PyBaMM#2483] [General]
Hi I have been playing around with pybamm and looking on how one would change the overall size of an electrode within a model to effect the C-Rate of the cell. 

Has anyone tried doing this? any example code?

Cheers in advance!

**Comment by RuiheLi** [comment]
The example code for changing parameter value is [here](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Getting%20Started/Tutorial%204%20-%20Setting%20parameter%20values.ipynb). The parameter you need to change will be "Electrode height [m]" and "Electrode width [m]", as the electrode surface is defined [here](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/parameters/geometric_parameters.py#:~:text=self.A_cc%20%3D%20self.L_y%20*%20self.L_z%20%20%23%20Current%20collector%20cross%20sectional%20area):

**Reply by RuiheLi** [reply]
Or more precisely, A_cc is the current collector surface area but is normally equal to electrode surface area
